packages/template/src/project.meta

```diff
@@ -4,7 +4,7 @@
     "background": "rgb(20,20,20)",
     "range": [
       0,
-      null
+      11.036440586280815
```

Comment:
Changes to the template package also shouldn't be committed unless necessary.

Reply:
Changed everything from `template package` back to their original values including

`template/src/project.meta`
```diff
"range": [
  0,
+ null
- 11.0364...
]
```

`template/src/scenes/example.meta`
```diff
"timeEvents": [
    {
      "name": "circle",
+    "targetTime": 3
-     "targetTime": 3.02....
    }
  ]
```